dapr/proto/runtime/v1/dapr.proto

```diff
@@ -870,12 +876,72 @@ message SubtleVerifyAlpha1Request {
   bytes signature = 5;
 }
 
-// SubtleVerifyAlpha1Response is the response for SubtleVerifyAlpha1.
-message SubtleVerifyAlpha1Response {
+// SubtleVerifyResponse is the response for SubtleVerifyAlpha1.
+message SubtleVerifyResponse {
   // True if the signature is valid.
   bool valid = 1;
 }
 
+// EncryptRequest is the request for EncryptAlpha1.
+message EncryptRequest {
+  // Request details. Must be present in the first message only.
+  EncryptRequestOptions options = 1;
+  // Chunk of data of arbitrary size.
+  common.v1.StreamPayload payload = 2;
+}
+
+// EncryptRequestOptions contains options for the first message in the EncryptAlpha1 request.
+message EncryptRequestOptions {
+  // Name of the component. Required.
+  string component_name = 1 [json_name="componentName"];
+  // Name (or name/version) of the key. Required.
+  string key_name = 2 [json_name="keyName"];
+  // Key wrapping algorithm to use. Required.
+  // Supported options include: A256KW (alias: AES), A128CBC, A192CBC, A256CBC, RSA-OAEP-256 (alias: RSA).
+  string algorithm = 3;
+  // Cipher used to encrypt data (optional): "aes-gcm" (default) or "chacha20-poly1305"
+  string cipher = 10;
```

Comment:
@ItalyPaleAle I think having these two might confuse the user. 
Should we rename the fields to make it clear? 
`keyWrappingAlgorithm` and `dataEncryptionCipher`?

Reply:
Yeah I can see that. I'll change this.